src/hooks/usePastTransaction/usePastTransaction.test.tsx

```diff
@@ -128,83 +124,87 @@ describe("usePastTransaction", () => {
   describe("fetch past transactions on initialisation", () => {
     it("should populate the past transactions with values", async () => {
       expect.assertions(1);
-      mockGetPastTransactions.mockReturnValueOnce(mockPastTransactions);
+      mockGetPastTransactions.mockReturnValue(mockPastTransactions);
 
       const { result, waitForNextUpdate } = renderHook(
-        () => usePastTransaction(id, key, endpoint),
+        () => usePastTransaction([id], key, endpoint),
         { wrapper }
       );
 
       await waitForNextUpdate();
 
-      expect(result.current.pastTransactionsResult).toStrictEqual({
-        pastTransactions: [
-          {
-            category: "tt-token",
-            quantity: 1,
-            identifierInputs: [
-              {
-                ...defaultIdentifier,
-                label: "first",
-                value: "AAA987654321"
-              }
-            ],
-            transactionTime: toDate(1596530356430)
-          },
-          {
-            category: "tt-token-lost",
-            quantity: 1,
-            identifierInputs: [
-              {
-                ...defaultIdentifier,
-                label: "first",
-                value: "AAA987654322"
-              }
-            ],
-            transactionTime: toDate(1596530356431)
-          },
-          {
-            category: "tt-token-lost",
-            quantity: 1,
-            identifierInputs: [
-              {
-                ...defaultIdentifier,
-                label: "first",
-                value: "AAA987654323"
-              }
-            ],
-            transactionTime: toDate(1596530356432)
-          }
-        ]
-      });
+      expect(result.current.pastTransactionsResult).toStrictEqual([
+        {
+          category: "tt-token",
+          quantity: 1,
+          identifierInputs: [
+            {
+              ...defaultIdentifier,
+              label: "first",
+              value: "AAA987654321"
+            }
+          ],
+          transactionTime: toDate(1596530356430)
```

Comment:
Noticed several uses of `toDate`, any idea what's the advantage of this over `new Date(...)`?

Reply:
I am not sure, copied this from other parts of the codebase 🙈  I guess it is consistent with our usage of `date-fns` everywhere?

Reply:
Yea i saw it then was like uhoh, can we stick to `new Date(...)` instead? 😊 
Can use `date-fns` only when necessary perhaps

Reply:
thanks!! (:

Reply:
Changed!